src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
+const populationItems = document.querySelectorAll('.population');
+const averagePopulationItem = document.querySelector('.average-population');
+const totalPopulationItem = document.querySelector('.total-population');
+const populationData = [];
 
-// write your code here
+populationItems.forEach(item => populationData.push(item.textContent));
+
+function parseNumber(str) {
+    return parseInt(str.replace(/,/g, ''), 10);
+}
+
+const populationNumbers = populationData.map(parseNumber);
+const totalNum = populationNumbers.reduce((num, sum) => num + sum, 0);
```

Comment:
it is possible to avoid forEach and map iterations.
try to make in reduce operations which you did in forEach and map.